axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -174,13 +265,46 @@ class TestPredator(TestFSMPlayer):
         'manipulates_source': False,
         'manipulates_state': False
     }
+    """
+    transitions = [
+                    (0, C, 0, D),  NO ROUTE
```

Comment:
they are not active on the first move because "initial_state" is set to one.  should initial state get set to 0?

> DOI:10.1109/CEC.2006.1688322.

i was wondering what that was!